touchforms/backend/xformserver.py

```diff
@@ -165,7 +165,7 @@ def handle_request(content, server):
         session_id = content['session-id']
 
     action = content['action']
-    logger.info('Received action %s for session %s' % (action, session_id))
+    logger.info('event=received action=%s session=%s' % (action, session_id))
```

Comment:
Just a general comment but often the way to do this kind of logging is to add the parameters to the 'extra' arg which allows you to configure the output format in the logging config.
```
details = {'action': action, 'session': session_id}
logger.info('reveived %s action for session %s', action, sessino_id, extra=details)
```

I would also have thought that a format that is just space separated keys would be easiest to parse:
```
{date} {session} {domain} {received / processed} {action}
```